src/ray/protobuf/common.proto

```diff
@@ -456,6 +456,31 @@ message StreamingGeneratorReturnIdInfo {
   bool is_plasma_object = 2;
 }
 
+message LeaseSpec {
+  bytes lease_id = 1;
+  bytes job_id = 2;
+  Address caller_address = 3;
+  TaskType type = 4;
+  bytes actor_id = 5;
+  bool is_detached_actor = 6;
+  bytes root_detached_actor_id = 7;
+  int64 max_actor_restarts = 8;
+  map<string, double> required_resources = 9;
+  map<string, double> required_placement_resources = 10;
+  SchedulingStrategy scheduling_strategy = 11;
+  map<string, string> label_selector = 12;
+  int64 depth = 13;
+  RuntimeEnvInfo runtime_env_info = 14;
+  repeated ObjectReference dependencies = 15;
+  int32 max_retries = 16;
```

Comment:
why do we care about `max_retries` and `max_actor_restarts` for a lease? is it used by oom killer?